.rakeTasks

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Settings><!--This file was automatically generated by Ruby plugin.
```

Comment:
what is this file? is this from your text editor? can you delete it?

Reply:
I have no idea. I was thinking the same thing. I'll just clear it out

Reply:
This file is now gone